src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -48,6 +49,11 @@ public class EditCommand extends Command {
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_PHONE_MISSING_DIGITS = "The phone number must contain at least one digit.";
+
+    public static final String MESSAGE_EDIT_PERSON_SUCCESS_WITH_WARNING = MESSAGE_EDIT_PERSON_SUCCESS
```

Comment:
Instead of purely saying it is discouraged without reasoning, it might be better to ask the user to double check if it was indeed intentional? 
- If not something like "a3112" will be allowed and could be reported as a bug.